onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -4208,19 +4208,54 @@ def aten_native_dropout_backward(
     raise NotImplementedError()
 
 
+@torch_op("aten::native_group_norm", trace_only=True)
 def aten_native_group_norm(
-    input: TensorType,
-    weight: Optional[TensorType],
-    bias: Optional[TensorType],
-    N: INT64,
-    C: INT64,
-    HxW: INT64,
-    group: int,
-    eps: float,
-) -> tuple[TensorType, TensorType, TensorType]:
+    input: TFloat,
+    weight: Optional[TFloat],
+    bias: Optional[TFloat],
+    N: INT64 = None,  # pylint: disable=unused-argument
+    C: INT64 = None,  # pylint: disable=unused-argument
+    HxW: INT64 = None,  # pylint: disable=unused-argument
+    group: int = None,
+    eps: float = None,
+) -> Tuple[TFloat, TFloat, TFloat]:
     """native_group_norm(Tensor input, Tensor? weight, Tensor? bias, SymInt N, SymInt C, SymInt HxW, int group, float eps) -> (Tensor, Tensor, Tensor)"""
 
-    raise NotImplementedError()
+    # Assert(weight is not None, and, bias is not None)
+    # Create weight_instance_norm and bias_instance_norm
+    weight_inst = op.Constant(value_floats=[1.0] * group)
+    bias_inst = op.Constant(value_floats=[0.0] * group)
+    # 0 in the shape list keeps dimension value unchanged, for InstanceNorm need
+    shape = op.Constant(value_ints=[0, group, -1])
```

Comment:
What happens when group is None?

Reply:
This must not be None. Set to 1 as default, will equal to LayerNorm.